core/embed/sys/smcall/stm32/smcall_numbers.h

```diff
@@ -92,7 +92,8 @@ typedef enum {
   SMCALL_TROPIC_ECC_KEY_GENERATE,
   SMCALL_TROPIC_ECC_SIGN,
 
-  SMCALL_BACKUP_RAM_READ_POWER_MANAGER_DATA,
-  SMCALL_BACKUP_RAM_STORE_POWER_MANAGER_DATA,
+  SMCALL_BACKUP_RAM_SEARCH,
```

Comment:
SEARCH SM call is unused - consider whether to leave it implemented

Reply:
not really, it takes just little of flash and no security concerns as far as i can tell. feel free to keep it